src/network/http_server.rs

```diff
@@ -671,6 +670,137 @@ pub struct OnChainEventRequest {
     reverse: Option<bool>,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub enum HubEventType {
```

Comment:
[Can we keep these consistent with hubs?](https://github.com/farcasterxyz/hub-monorepo/blob/be0e240eb9ebd3bd66e8751d6789096ed57242a7/protobufs/schemas/hub_event.proto#L7)